affiliation/dto.go

```diff
@@ -0,0 +1,115 @@
+package affiliation
```

Comment:
Why do we need affiliation? My understanding is that V2 is only outputting "identities" (not even profiles/users). V2 is not calling any affiliations APIs nor is calculating org name(s)?
not a stopper - just wondering.